### DISCUSSION TITLE
Outdated info on "What can I do when I run into build output " guide

### DISCUSSION THREAD

**Discussion by guitheengineer** [vercel/community#1298] [Help] [answered]
### Summary
https://vercel.com/guides/what-can-i-do-when-i-run-into-build-output-limits-with-next-js-on-vercel

This last paragraph is outdated, `fallback: 'blocking'` is not experimental anymore.
> If you prefer to delay the entire response until your page has been fully rendered, you can use the experimental blocking option by returning fallback: 'unstable_blocking' instead of fallback: true.

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this! I updated the guide
